### PR TITLE
[bluescarni-tanuki] new port

### PR DESCRIPTION
**Pull request by nickdademo**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] The name of the port matches an existing name for this component on https://repology.org/ if possible, and/or is strongly associated with that component on search engines.
- [ ] ~~Optional dependencies are resolved in exactly one way. For example, if the component is built with CMake, all `find_package` calls are REQUIRED, are satisfied by `vcpkg.json`'s declared dependencies, or disabled with [CMAKE_DISABLE_FIND_PACKAGE_Xxx](https://cmake.org/cmake/help/latest/variable/CMAKE_DISABLE_FIND_PACKAGE_PackageName.html).~~
- [ ] ~~The versioning scheme in `vcpkg.json` matches what upstream says.~~
- [x] The license declaration in `vcpkg.json` matches what upstream says.
- [x] The installed as the "copyright" file matches what upstream says.
- [x] The source code of the component installed comes from an authoritative source.
- [x] The generated "usage text" is accurate. See [adding-usage](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/examples/adding-usage.md) for context.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is in the new port's versions file.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Review by LilyWangLL — DISMISSED** [review]

**Review by BillyONeal — COMMENTED** [review]
I am concerned that when I search for this I also get hits for https://github.com/nodchip/tanuki- leading to potential for confusion. Would you find it acceptable if the port is named `bluescarni-tanuki` instead?

The rest of the port seems OK.

File: ports/tanuki/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO bluescarni/tanuki
+    REF 14e076abf879bcded0cc437cd09f3766969b15d1
+    SHA512 e847e13e757aa2eee0ed8cde584d39545786a233905f6ed30110dcfc325dfe26eeb37ee00bff4936aa311bfdd66bb3f3f58f41aafc021701c4ea056caf964ee3
+    HEAD_REF main
+)
+
+file(INSTALL ${SOURCE_PATH}/include/tanuki/tanuki.hpp DESTINATION ${CURRENT_PACKAGES_DIR}/include)
```
Comment (BillyONeal) [inline ports/tanuki/portfile.cmake:9]:
```suggestion
file(INSTALL "${SOURCE_PATH}/include/tanuki/tanuki.hpp" DESTINATION ${CURRENT_PACKAGES_DIR}/include)
```

**Comment by nickdademo** [comment]
> I am concerned that when I search for this I also get hits for https://github.com/nodchip/tanuki- leading to potential for confusion. Would you find it acceptable if the port is named `bluescarni-tanuki` instead?
> 
> The rest of the port seems OK.

Alright! Will make the change later today.

**Comment by BillyONeal** [comment]
Thanks for the new port!

**Pull request merged**